Sources/CloudKitSyncMonitor/SyncMonitor.swift

```diff
@@ -178,6 +178,28 @@ public class SyncMonitor: ObservableObject {
                 return "icloud.slash"
             }
         }
+        
+        // A text you could use to disay the status
```

Comment:
```suggestion
        // A string you could use to display the status
```

Thanks for the PR! Can you make the tweak above (calling it a "string" instead of "text", and fixing the typo in "display")?

Other than that, it's great - I had to think, and look, 'cause I could swear I already had that functionality somehow, but I can't find it.

Reply:
Could be a useful addition, now it's there :)